util/metrics/src/service_metrics.rs

```diff
@@ -137,7 +144,7 @@ impl ServiceMetrics {
         }
     }
 
-    /// Track GRPC message size
+    /// Tracks gRPC message name and corresponding message size into a histogram
```

Comment:
It's quite possible I'm missing it in the inner workings but the "into a histogram" part doesn't seem like it is happening here.  This seems like an _input_ and eventually when one wants the output that might be a _histogram_.

Reply:
This call message call tracks the message size and ships it to a Prometheus histogram